LOAN/LoanContracts/LoanApplications.rdf

```diff
@@ -132,6 +298,51 @@
 		<skos:definition xml:lang="en">credit rating that provides an opinion of the creditworthiness of a person</skos:definition>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-loan-ln-app;LoanApplication">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-com;Application"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;comprises"/>
+				<owl:onClass rdf:resource="&fibo-loan-ln-ln;CombinedLoanToValueRatio"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;comprises"/>
+				<owl:onClass rdf:resource="&fibo-loan-ln-ln;LoanToValueRatio"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;comprises"/>
+				<owl:someValuesFrom rdf:resource="&fibo-loan-ln-app;AllBorrowersMonthlyIncome"/>
```

Comment:
I agree, but see above - I just moved this stuff out of LoanCore and into this ontology so that we didn't lose what had been done, but it needs a lot more review by SMEs.